### PR TITLE
Rework guild commands

### PR DESCRIPTION
**Pull request by n1tr0-5urf3r**
- guild log shows more entries
- added emojis to guild info

### CODE REVIEW

File: guildwars2/guild/general.py [hunk 1]
```diff
@@ -43,14 +44,27 @@ class GeneralGuild:
...
+            value='{} {}'.format(
+                self.get_emoji(ctx, "aetherium"), results["aetherium"]))
+        data.add_field(
+            name="Resonance",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "resonance"), results["resonance"]))
+        data.add_field(
+            name="Favor",
+            value='{} {}'.format(
+                self.get_emoji(ctx, "favor"), results["favor"]))
```
Comment (Maselkov) [inline guildwars2/guild/general.py]: This can be done with a for loop and a list of those currencies

File: guildwars2/guild/general.py [hunk 2]
```diff
@@ -65,8 +79,7 @@ class GeneralGuild:
     @guild.command(name="members", usage="<guild name>")
     @commands.cooldown(1, 20, BucketType.user)
     async def guild_members(self, ctx, *, guild_name=None):
-        """Get list of the first 20 members and their ranks.
-        Only displays the highest ranks.
+        """Get list of members and their ranks.
```
Comment (Maselkov) [inline guildwars2/guild/general.py]: Maybe change it to "Shows a list of members and their ranks" for consistency

File: guildwars2/guild/general.py [hunk 3]
```diff
@@ -140,35 +157,39 @@ class GeneralGuild:
...
+            colour=await self.get_embed_color(ctx))
         data.set_author(name=guild_name.title())
-        counter = 0
         item_counter = 0
         amount = 0
+        lines = []
+        length_lines = 0
         itemlist = []
         for item in treasury:
             res = await self.db.items.find_one({"_id": item["item_id"]})
```
Comment (Maselkov) [inline guildwars2/guild/general.py]: You can use `await self.fetch_item(id)`

File: guildwars2/guild/general.py (hunk 1 above)
Comment (n1tr0-5urf3r) [reply guildwars2/guild/general.py]: Changed to a for loop

File: guildwars2/guild/general.py (hunk 2 above)
Comment (n1tr0-5urf3r) [reply guildwars2/guild/general.py]: Changed helper message

File: guildwars2/guild/general.py (hunk 3 above)
Comment (n1tr0-5urf3r) [reply guildwars2/guild/general.py]: Changed to await self.fetch_item(id)

**Pull request merged**